### ISSUE TITLE
Multiple Jobs?

### ISSUE THREAD

**Issue by naisanza** [ansible/awx#262]
##### ISSUE TYPE

 - Feature Request

##### COMPONENT NAME

 - UI

##### SUMMARY

From the default build, it seems to queue jobs one after another, and if one job is IO blocking, the other jobs never start. 

Is there an option I've missed going through the documentation on how to allow multiple jobs run simultaneously?

Are each signed in user allowed a single job operation? If that's so, I could log into multiple users to run multiple jobs.

##### ENVIRONMENT

* AWX version: ece93c45c9c932cfbfba2b44e217e99a6bb6a752
* Ansible version:  2.4.0.0
* Operating System: Proxmox Ubuntu 16.04 server
* Web Browser: Chrome

**Comment by matburt** [comment]
There is an option on job templates to allow multiples of the same job template to run at the same time.

Other than you need to make sure you have enough capacity to run simultaneous jobs: http://docs.ansible.com/ansible-tower/latest/html/userguide/jobs.html#job-concurrency

**Comment by akcrisp** [comment]
@matburt Can I check something,.... So we are seeing on awx 1.1.173 build, jobs queue. 

I have tried some tests, so now need someone to help clarify behaviour...

Host running the awx container has 16 gig ram... So i don't think we are hitting a memory limitation.

We have 2 job templates, both were using the same project and we had update on launch ticked - so thought that was the problem. So I've now created another project but pointing at the same stash repo (again with update on launch not ticketed for either - does awx treat these independently or does it know they are both pointing at the same stash repo ??.)

So having configured both job templates with "independent projects ie different project names but pointing at the same stash repo" we are still seeing them queue. One job has 10 hosts in the inventory, the other 4 hosts in the inventory.  

Andy
